File: net/pdfjet/A5.cs
/**
 *  A5.cs
 *
Copyright 2023 Innovatics Inc.

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.
*/

using System;


namespace PDFjet.NET {
/**
 *  Used to specify PDF page with size <strong>A5</strong>.
 *  For more information about the page size classes - A3, A4, A5, B5, Executive, Letter, Legal and Tabloid - see the Page class.
 *
 */
public class A5 {
    public static readonly float[] PORTRAIT = new float[] {420.0f, 595.0f};
    public static readonly float[] LANDSCAPE = new float[] {595.0f, 420.0f};
}
}   // End of namespace PDFjet.NET

File: com/pdfjet/A5.java
/*
 * A5.java
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
package com.pdfjet;

/**
 * Used to specify PDF page with size <strong>A5</strong>.
 * For more information about the page size classes - A3, A4, A5, B5, JISB5, Executive, Letter, Legal and Tabloid - see the Page class.
 */
public class A5 {
    private A5() {
    }
    /**
     * This is a public static variable that specifies that page size in portrait orientation.
     */
    public static final PageSize PORTRAIT = new PageSize(420.0f, 595.0f);
    /**
     * This is a public static variable that specifies that page size in landscape orientation.
     */
    public static final PageSize LANDSCAPE = new PageSize(595.0f, 420.0f);
}
